kubespawner/spawner.py

```diff
@@ -850,8 +877,8 @@ def get_env(self):
         env = super(KubeSpawner, self).get_env()
         env.update({
             'JPY_USER': self.user.name,
-            'JPY_COOKIE_NAME': self.user.server.cookie_name,
-            'JPY_BASE_URL': self.user.server.base_url,
+            'JPY_COOKIE_NAME': self.server.cookie_name,
+            'JPY_BASE_URL': self.server.base_url,
```

Comment:
I think this entire `.get_env()` method can be deleted. It's been deprecated for a while and is not needed by recent images.

Reply:
That should probably be a different PR though. Agree otherwise.